sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -446,3 +455,51 @@ def _x_else_x_o(self, x: Optional[Array]) -> Array:
             raise ValueError("Context x needed when not preconditioned to x_o.")
         else:
             return self.x_o
+
+    @staticmethod
+    def _check_for_multiple_x(x: Tensor) -> None:
```

Comment:
Single line docstring. Especially if we want to leave the name of the function like this.